yogstation/code/game/gamemodes/battle_royale/loot.dm

```diff
@@ -1,196 +1,306 @@
-					new /obj/item/ammo_casing/caseless/rocket(src)
-					return
-				if(8)
-					new /obj/item/gun/ballistic/automatic/sniper_rifle(src)
-					return
-				if(9)
-					new /obj/item/gun/ballistic/automatic/ar(src)
-					return
-				if(10)
-					new /obj/item/gun/ballistic/automatic/m90/unrestricted(src)
-					return
-	else //Item crate, but you still get a gun anyway
-		new /obj/item/gun/ballistic/automatic/pistol(src)
-		if(prob(30)) //Common
-			new /obj/item/storage/firstaid(src)
-			new /obj/item/ammo_box/a357(src)
-			new /obj/item/ammo_box/c38(src)
-			new /obj/item/clothing/suit/armor/vest(src)
-			return
-		else if(prob(30))
-			new /obj/item/ammo_box/c9mm(src)
-			new /obj/item/ammo_box/c10mm(src)
-			new /obj/item/ammo_box/c45(src)
-			new /obj/item/clothing/suit/armor/vest(src)
-			new /obj/item/reagent_containers/autoinjector/mixi(src)
-			new /obj/item/storage/firstaid(src)
-			new /obj/item/shield/riot(src)
-			return
-		else if(prob(25))
-			new /obj/item/ammo_box/a40mm(src)
-			new /obj/item/ammo_box/a762(src)
-			new /obj/item/reagent_containers/autoinjector/mixi(src)
-			new /obj/item/reagent_containers/autoinjector/derm(src)
-			new /obj/item/storage/firstaid/fire(src)
-			new /obj/item/clothing/suit/armor/riot(src)
-			return
-		else if(prob(10))
-			new /obj/item/ammo_box/no_direct/n762(src)
-			new /obj/item/ammo_box/foambox/riot
-			new /obj/item/clothing/suit/space/hardsuit/ert/sec(src)
-			new /obj/item/storage/firstaid(src)
-			new /obj/item/storage/firstaid/fire(src)
-			return
-		else //Legendary
-			new /obj/item/ammo_box/a357(src)
-			new /obj/item/ammo_box/c38(src)
-			new /obj/item/ammo_box/c9mm(src)
-			new /obj/item/ammo_box/c10mm(src)
-			new /obj/item/ammo_box/c45(src)
-			new /obj/item/ammo_box/a40mm(src)
-			new /obj/item/ammo_box/a762(src)
-			new /obj/item/ammo_box/no_direct/n762(src)
-			new /obj/item/ammo_box/foambox/riot
-			new /obj/item/clothing/suit/space/hardsuit/syndi(src)
-			new /obj/item/storage/firstaid(src)
-			new /obj/item/storage/firstaid/toxin(src)
-			return
-	new /obj/item/storage/toolbox/mechanical(src) //if by some fucking miracle it didn't spawn anything, give them the best weapon of all.
+	var/type = rand(1,4)//for a couple different themes
+	
+	switch(type)//it's in two blocks so healing fountains still get reskinned but don't get items
+		if(1)//weapon focus (to fuel the fight)
+			name = "Weapons Supply Crate"
+			add_atom_colour(LIGHT_COLOR_BLOOD_MAGIC, FIXED_COLOUR_PRIORITY)
+		if(2)//armour focus (so people can select what they want)
+			name = "Armour Supply Crate"
+			add_atom_colour(LIGHT_COLOR_BLUE, FIXED_COLOUR_PRIORITY)
+		if(3)//allrounder
+			name = "Misc Supply Crate"
+			add_atom_colour(LIGHT_COLOR_YELLOW, FIXED_COLOUR_PRIORITY)
+		if(4)//KABOOOM AHAHAHAHAHA (better hope the armour is explosion resistant)
+			name = "Utility Supply Crate"
+			add_atom_colour(LIGHT_COLOR_PURPLE, FIXED_COLOUR_PRIORITY)
+		if(5)//https://www.youtube.com/watch?v=Z0Uh3OJCx3o
```

Comment:
the switch only goes to 4, this is 5

Reply:
Ah, that explains things, i initially just had 3, then i upped to 4, and then 5